site/docs/v1/tech/apis/_system-logs-request-body.adoc

```diff
@@ -6,6 +6,11 @@ The format string used to format the date and time columns in the export result.
 +
 When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.
 
+[field]#includeArchived# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
```

Comment:
On request bodies, when there is something new, we include the `[since]` field. For example: `[since]#Available since 1.30.0#` if it was added in version 1.30.0`. 

You can see other examples here: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/docs/v1/tech/apis/_user-request-body.adoc